.github/issue_template.md

```diff
@@ -0,0 +1,10 @@
+# If your issue looks like hardware fault or a configuration problem this may is the wrong place to post to.
+## Please consider to use [RCGroups](rcgroups.com/forums/showthread.php?t=2464844), Slack or any other forum before posting here or even better ask the manufacturer.
```

Comment:
'Please consider using'. Also, we should probably add 'Slack [register here](http://www.betaflight.tk/)' and 'the manufacturer of the hardware you are using'.